src/shared/components/FluxMonacoEditor.tsx

```diff
@@ -46,7 +46,6 @@ const FluxEditorMonaco: FC<Props> = ({
 }) => {
   const lspServer = useRef<LSPServer>(null)
   const [editorInst, seteditorInst] = useState<EditorType | null>(null)
-  const [docVersion, setdocVersion] = useState(2)
```

Comment:
I removed this because it doesn't seem to be used anywhere other than the increment inside the `onChange`. The increment inside the `onChange` was causing some weird behavior where only the last character typed was recorded once I introduced the `debouce`.

Reply:
i think this is needed by the LSP for some reason. reach out to @scbrickley to make sure i'm not just blowing bubbles

Reply:
Slack with Sean: https://influxdata.slack.com/archives/C0AD7FFS9/p1614795598154600

We're keeping track of the document version in the monoco server so we don't need to additionally track it in the flux editor component. 
https://github.com/influxdata/ui/blob/master/src/external/monaco.flux.server.ts#L278